test/extensions/clusters/aggregate/cluster_test.cc

```diff
@@ -139,7 +139,7 @@ class AggregateClusterTest : public Event::TestUsingSimulatedTime, public testin
   Upstream::LoadBalancerFactorySharedPtr lb_factory_;
   Upstream::LoadBalancerPtr lb_;
   Upstream::ClusterTrafficStatNames stat_names_;
-  Upstream::ClusterTrafficStats stats_;
+  std::unique_ptr<Upstream::ClusterTrafficStats> stats_;
```

Comment:
use the nickname.

I think you need to recursive grep for this patten to make sure you get all of them, with and without the `Upstream::`